data-collection/src/main/java/refactoringml/ProcessMetricsCollector.java

```diff
@@ -215,7 +222,8 @@ private void updateProcessMetrics(RevCommit commit, RevCommit commitParent) thro
 		for (Long yesId : allYeses) {
 
 			Yes yes = db.findYes(yesId);
-
+			//TODO: the filePath from yes and Diffentry do not always match, e.g.
```

Comment:
maybe we can clean the one in the `Yes` up? After all, we don't care about the full path in the fie system, but the relative one